LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright © 2017-present, [Encode OSS Ltd](http://www.encode.io/).
+Copyright © 2017-present, [Encode OSS Ltd](https://www.encode.io/).
```

Comment:
I think there's no problem on updating this in the license, but just to be sure... Do you know if this is a problem @tomchristie ?

Reply:
It's fiiiiiiiiiiiiiiiinnnnnneeeeeeeeeeeeeeee

Reply:
Tho unrelated to the rest of the PR. Which. Hrm.

Reply:
If we update that link, let's have it as a separate PR, from someone on the team.